content/docs/start/visualization.md

```diff
@@ -0,0 +1,108 @@
+# Visualization with DVC Plots
+
+In this section, we'll add visualization to [`example-dvc-experiments`][ede]
+project. If you would like to try these yourself, please refer to the project
+[README] about how to install.
+
+[ede]: https://github.com/iterative/example-dvc-experiments
+[readme]:
+  https://github.com/iterative/example-dvc-experiments/blob/main/README.md
+
+One of the important plots that shows the classification performance is the
+confusion matrix. DVC uses Vega-Lite to plot the text files. In order to produce
+a confusion matrix, Vega expects a CSV file in the form:
+
+```csv
+actual,predicted
+0,0
+0,2
+...
+```
+
+We added a [straightforward loop] to the project to generate this file from the
+predictions.
+
+[straightforward loop]:
+  https://github.com/iterative/example-dvc-experiments/blob/main/src/train.py#L213
+
+After running the experiment with `dvc exp run`, DVC will produce
+`plots/confusion.csv` file that can immediately be shown in the browser:
+
+```dvc
+$ dvc plots show plots/confusion.csv --template confusion -x actual -y predicted
+file:///Users/.../example-dvc-experiments/plots/confusion.json.html
+```
+
+The file contains the following plot that shows the performance of the CNN
+classifier:
+
+![confusion matrix](/img/start_visualization_confusion1.png)
+
+Let's produce another plot to see misclassified examples from each class. This
+procedure is a bit more complex than creating a CSV file but you can see the
+code in [Github][misclassified-example-code]
+
+[misclassified-example-code]:
+  https://github.com/iterative/example-dvc-experiments/blob/main/src/train.py#L58
```

Comment:
The linked example seems to create a CSV file. So what do we mean by "it's a bit more complex" than that?

Probably no need to use these potentially scary terms anyway. I'd just link to the code and be more plain about it. Something like "Here's how the CSV plot file is generated" -- ideally as a note like the previous link to the code.

Reply:
But should we show a sample of what the end-result (CSV file) looks like?

Reply:
It's not a CSV file, it's an image generated on-the-fly from other images so a bit more involved than creating a CSV file.